src/assets/global.css

```diff
@@ -96,3 +99,11 @@ li::marker {
 .plaque h1 {
     color: var(--color-accent);
 }
+
+.warning {
```

Comment:
If I ever need more than this, consider splitting the semantic color information from the shape of this div. Or better yet, make a component for this.